examples/intro_data_profiler.ipynb

```diff
@@ -0,0 +1,442 @@
+   "cell_type": "markdown",
+   "id": "4054a3df",
+   "metadata": {},
+   "source": [
+    "## Basic examples"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fc7083d8",
+   "metadata": {},
+   "source": [
+    "This section shows the basic example of DataProfiler. A CSV dataset is read using the data reader, then the returned data is given to the DataProfiler to obtain the properties and stastistics."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7a75da26",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# use data reader to read input data\n",
+    "#data = dp.Data(\"../dataprofiler/tests/data/csv/aws_honeypot_marx_geo.csv\")\n",
+    "data = dp.Data(os.path.join(data_path, \"csv/aws_honeypot_marx_geo.csv\"))\n",
+    "print(data.data.head())\n",
+    "\n",
+    "# run data profiler and get the report\n",
+    "profile = dp.Profiler(data)\n",
+    "report  = profile.report(report_options={\"output_format\":\"compact\"})\n",
+    "\n",
+    "# print the report\n",
+    "print(json.dumps(report, indent=4))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9e0ffdf6",
+   "metadata": {},
+   "source": [
+    "The report includes `global_stats` and `data_stats` for the given dataset. The former contains overall properties of the data such as number of rows/columns, null ratio, duplicate ratio, while the later contains specific properties and statistics for each column such as min, max, mean, variance, etc. In this example, the `compact` format of the report is used to shorten the full list of the results. To get more results related to detailed predictions at the entity level from the DataLabeler component or histogram results, the format `pretty` should be used."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "81381be5",
+   "metadata": {},
+   "source": [
+    "In addition to reading the input data from multiple file types, DataProfiler allows the input data as a dataframe."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c3519d5d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# run data profiler and get the report\n",
+    "my_dataframe = pd.DataFrame([[1, 2.0],[1, 2.2],[-1, 3]], columns=[\"col_int\", \"col_float\"])\n",
+    "profile = dp.Profiler(my_dataframe)\n",
+    "report  = profile.report(report_options={\"output_format\":\"compact\"})\n",
+    "\n",
+    "# Print the report\n",
+    "print(json.dumps(report, indent=4))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a6bf9d1",
+   "metadata": {},
+   "source": [
+    "## Profiler options"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "97551a0c",
+   "metadata": {},
+   "source": [
+    "DataProfiler can run several selected components if needed. For example, if the users only want the statistics information, they may turn off the DataLabeler functionality. Below, let's remove the histogram and data labeler component while running DataProfiler."
```

Comment:
This is a pretty short section for profile options, I think this could be expanded